CMakeLists_files.cmake

```diff
@@ -228,6 +229,7 @@ if(ENABLE_ECL_OUTPUT)
           tests/test_writenumwells.cpp
           tests/test_Solution.cpp
           tests/test_regionCache.cpp
+          tests/test_calc_cellvol.cpp
```

Comment:
Just noticed: this test program does not have any ecl dependencies, and the code that it tests is always compiled (irrespective of ENABLE_ECL_OUTPUT), so the test should be unconditionally included as well, by moving this to approx line 144.